src/PrestaShopBundle/Controller/Admin/MultistoreController.php

```diff
@@ -105,6 +108,8 @@ public function header(bool $lockedToAllShopContext): Response
             'isAllShopContext' => $isAllShopContext,
             'isGroupContext' => $this->multistoreContext->isGroupShopContext(),
             'lockedToAllShopContext' => $lockedToAllShopContext,
+            'shouldDisplayColorNotification' => !$isAllShopContext && empty($currentContext->getColor()),
```

Comment:
Both fields feel a bit redundant You could only fill the `colorConfigLink` when `empty($currentContext->getColor())` so that you only need to rely one this one variable in the template

Reply:
:heart_eyes: on you and your family. God saves Jo.